library/templates/sca_e_projections.yml

```diff
@@ -0,0 +1,32 @@
+dataset:
+  name: &name sca_e_projections
```

Comment:
I think it might make sense if we tested one of these datasets on different machine? Do you have the raw files handy and want to send one over?